packages/graphback-cli/src/helpers/db.ts

```diff
@@ -4,13 +4,26 @@ import { GraphQLBackendCreator, PostgresSchemaManager } from 'graphback';
 import { logError, logInfo } from '../utils'
 import { checkDirectory } from './common'
 
+const handleError = (err: { code: string; message: string; }): void => {
+  if(err.code === 'ECONNREFUSED') {
+    logError('Database not running. Run docker-compose up -d or docker-compose start to start the database.')
+  } else {
+    logError(err.message)
+  }
+  process.exit(0)
+}
+
 export const dropDBResources = async(): Promise<void> => {
-  const { dbConfig } = JSON.parse(readFileSync(`${process.cwd()}/config.json`, 'utf8'))
-  const manager = new PostgresSchemaManager(dbConfig);
-  // tslint:disable-next-line: await-promise
-  await manager.getConnection().raw('DROP SCHEMA public CASCADE;')
-  // tslint:disable-next-line: await-promise
-  await manager.getConnection().raw('CREATE SCHEMA public;')
+  try {
+    const { dbConfig } = JSON.parse(readFileSync(`${process.cwd()}/config.json`, 'utf8'))
+    const manager = new PostgresSchemaManager(dbConfig);
+    // tslint:disable-next-line: await-promise
+    await manager.getConnection().raw('DROP SCHEMA public CASCADE;')
```

Comment:
👍 This is what we need for the moment.
Problem with this is that this is postgres specific and public schema may not exist but let's not bother with that for the moment. This code will be replaced soon.
Some warning message here will be cool - your data is lost. 
Do not push that to production etc.